### ISSUE TITLE
Prediction with external dataset 

### ISSUE THREAD

**Issue by JuanFuriaz** [tersekmatija/eWaSR#2]
Hi everyone!

I'm trying to run the `prediction.py` as shown in here: 
```bash
python predict.py \
--dataset_config configs/examples.yaml \
--model ewasr_resnet18 \
--weights path/to/model/weights.pth \
--output_dir output/predictions
```
I downloaded the weights and reference the path to my images in `configs/examples.yaml`. Nevertheless, I can not run the model with my own images in contrast to the `wasr` repository. 

Could it be that in lines 59-65 of the function `predict` from `predict.py` there isn't any option for external datasets? 

Thanks in advance!

**Comment by tersekmatija** [comment]
@JuanFuriaz 
Thanks, should be fixed in the latest commit. It should work with example config. Note that it will expect ``image_dir` and `imu_dir` where IMU masks have the same name as images, but end with `.png`.

**Comment by JuanFuriaz** [comment]
Hi @tersekmatija,

Code worked! Thanks! 

Nevertheless, I needed to make a workaround because I don't have any IMU data for my own dataset. I just copy one file of the examples and duplicated for all my images. Is it planned to work also without input of IMU data, as in WASR? Or should I close the issue?

In order to work with custom data, I modified in the `predict.py` parser the field required to `false`:   `parser.add_argument("--dataset", type=str, choices=["mastr", "mods"], required=False)`. 

Best regards!

**Comment by tersekmatija** [comment]
@JuanFuriaz I haven't tested the `predict.py` on a dataset without the IMU. I think some small changes will have to be made for such dataset and inference on eWaSR without the IMU mask. Feel free to make a PR with a fix for #3 as well :)
